README.md

```diff
@@ -1,39 +1,38 @@
-**Table of Contents**
 # Automated setup and configuration for most of OpenAustralia Foundation's servers
 
 <!-- vscode-markdown-toc -->
```

Comment:
This TOC was generated with https://marketplace.visualstudio.com/items?itemName=joffreykern.markdown-toc. It doesn't make sense to me to manually edit this - it's just going to be created again in the same way and these changes will be reverted. I ]think it makes more sense to have the linter ignore this block, or perhaps to switch to some other TOC generator that matches the linter's preferences.